### PR TITLE
MAHOUT-2122 Add Getting Started Docker

### PR DESCRIPTION
**Pull request by rawkintrevo**
### Purpose of PR:
Please give a short description of what this PR is for.
Creating a Docker container to help people get started on Mahout

### Important ToDos
Please mark each with an "x"
- [x] A JIRA ticket exists (if not, please create this first)[https://issues.apache.org/jira/browse/mahout/]
- [x] Title of PR is "MAHOUT-XXXX Brief Description of Changes" where XXXX is the JIRA number.
- [x] Created unit tests where appropriate
- [x] Added licenses correct on newly added files
- [x] Assigned JIRA to self
- [x] Added documentation in scala docs/java docs, and to website
- [x] Successfully built and ran all unit tests, verified that all tests pass locally.
- [x] Docs,

Does this change break earlier versions?
No

Is this the beginning of a larger project for which a feature branch should be made?
No

### CODE REVIEW

**Comment by rawkintrevo** [comment]
Someone help me. I've made a mess on my git merging...

The docker file is available at `apache/mahout-zeppelin:14.1`

**Comment by rawkintrevo** [comment]
Hooray for self help! I fixed it, w00t w00t. 

@andrewmusselman or whoever else- please check the docs for grammar, bc that is a really weak suit for me. You can test the produced docker image with 

```
docker pull apache/mahout-zeppelin:14.1
docker run -p 8080:8080 --rm --name whatever apache/mahout-zeppelin:14.1
```

And then surf to http://localhost:8080 in your favorite browser.

File: docker/getting-started/readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+```
+export CONTAINER_NAME=$USER/mahoutgui
+export VERSION=14.3-SNAPSHOT
+```
+
+## Building Container
+
+```bash
+docker build -t $CONTAINER_NAME:$VERSION .
```
Comment (andrewmusselman) [inline docker/getting-started/readme.md:13]: Is this not `pull`

File: website/_posts/2020-10-18-version-14-2-released.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: 14.2 Is Released!
```
Comment (andrewmusselman) [inline website/_posts/2020-10-18-version-14-2-released.md:3]: This is 14.1

File: docker/getting-started/readme.md (hunk 1 above)
Comment (andrewmusselman) [reply docker/getting-started/readme.md:13]: Nevermind I see the "how to pull" doc below

**Pull request merged**